src/map/lua/lua_baseentity.cpp

```diff
@@ -12193,6 +12193,48 @@ uint32 CLuaBaseEntity::getMobFlags()
     return 0;
 }
 
+/************************************************************************
+ *  Function: setPetFlags()
+ *  Purpose : Manually set Pet flags
+ *  Example : player:setPetFlags(flags)
+ *  Notes   :
+ ************************************************************************/
+
+void CLuaBaseEntity::setPetFlags(uint32 flags)
```

Comment:
Do we need these new bindings? When we could just have:
```lua
local pet = master:getPet()
if pet then
    pet:setFlags(...)
end
```

Reply:
This does not work for pets as is, would have been easier but not a viable solution in its current form,
as montioned in a prevous comment, hopefully a refactor of the entity flags will provide a solution.